src/decrypters/widevineandroid/WVDecrypter.h

```diff
@@ -55,7 +55,8 @@ class ATTR_DLL_LOCAL CWVDecrypterA : public DRM::IDecrypter
 
   virtual void GetCapabilities(std::shared_ptr<Adaptive_CencSingleSampleDecrypter> decrypter,
                                const std::vector<uint8_t>& keyId,
-                               DRM::DecrypterCapabilites& caps) override;
+                               DRM::DecrypterCapabilites& caps,
+                               DRMMediaType mediaType) override;
```

Comment:
android compilers fails
this due to inappropriate/missing changes, this should be:

```suggestion
                               DRM::DRMMediaType mediaType) override;
```

and also its missing relative method code change on .cpp implementation

Reply:
after fix this, let me know if you want to merge PR now or wait